.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,7 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/pang
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pangenome/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/pangenome _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
+- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
```

Comment:
I assume this is a template change, but is pangenome ready for `nf-test` yet?

Reply:
This is a template change. I did not update tests so they can be run with nf-test. Let's keep this open for next week's hackathon. I will open an issue with details and hopefully someone will take care of this ;)